File: Hybrid_Task/Python/power_of_number.py
number = 16

for divisor in range(1,number + 1):
	power = number ** (1 / divisor)
	for base in range(2,number+1):
		if number % base == 0:
			if type(power) != float:
				print(count)
				print(divisor)

			

File: Hybrid_Task/Java/Breakfast/PowerOfNumber.java
public class PowerOfNumber {
public static void main(String[] args) {

int number = 4;
int exponent = 3;
int answer = 1;

for(int count = 0; count < exponent; count++) {
	answer *= number;
}

System.out.print(answer);
}
}
			